cli/tools/standalone.rs

```diff
@@ -209,6 +223,18 @@ async fn create_standalone_binary(
   Ok(final_bin)
 }
 
+fn set_windows_binary_to_gui(mut bin: Vec<u8>) -> Vec<u8> {
```

Comment:
I feel like this could use some code comments including a documentation link. I think I can tell what you're doing, flipping a few bits which are probably flags on the header but I gotta say it does look pretty magical, it would be nice to know what is happening precisely.

For example whate are the numbers 60-64 and 92? It may even be nice to use the windows flag constants for these values.